Objects/Objects/Geometry/BrepTrim.cs

```diff
@@ -32,6 +32,8 @@ int endIndex
     IsoStatus = isoStatus;
     TrimType = trimType;
     IsReversed = reversed;
+    StartIndex = startIndex;
```

Comment:
Why were we not doing this before?

Reply:
I'm not sure, @AlanRynne was there a reason you can remember for not setting the `StartIndex` and `EndIndex` props even though they are arguments passed to the constructor?

Reply:
I find no reason why we would ever do this, I'll dig in a bit more after standup.

Reply:
I double checked the usages of this constructor and it is definitely a bug/oversight.

This does not affect conversions for the most part, as the start/end vertex index is rarely used... but I'm sure this would fix some odd Brep edge-cases we may have been unaware of.

It will also ensure breps always have the same id (as missing to set start/end vertex would in fact result in a different id for that brep)